BuyPolish/Pola/SupportingFiles/pl.lproj/Localizable.strings

```diff
@@ -10,6 +10,7 @@
 "Report" = "Zgłoś";
 "photos:" = "zdjęcia:";
 "description (optional):" = "opis (opcjonalny):";
+"description:" = "opis:";
```

Comment:
Text wyżej do usunięcia.

Reply:
Nie traktujemy tego jako czegoś przejściowego? Dlatego nie usuwałem starych tłumaczeń.

Reply:
Ale jak zostawimy, to jak ktoś będzie robił to doda nowy bo nie pomyśli, że już jest taki tekst, a nie ma pewności, że ktoś przy PR będzie o tym pamiętał. Ale możemy zrobić to jeszcze inaczej. W zależności czy jest włączona możliwość wysyłanie zdjęć czy nie, wyświetlajmy tekst z optional czy nie.